packages/mdc-drawer/persistent/mdc-persistent-drawer.scss

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "@material/rtl/mixins";
+@import "@material/theme/mixins";
+@import "../mixins";
+@import "../slidable/mixins";
+@import "../slidable/variables";
+
+$mdc-persistent-drawer-width: 240px;
+
+.mdc-persistent-drawer {
+  @include mdc-drawer-base_;
+  width: 0;
+
+  &__drawer {
+    @include mdc-theme-prop(background, background);
+    @include mdc-rtl-reflexive-box(border, right, 1px solid #e4e4e4);
+    @include mdc-rtl-reflexive-position(left, 0);
+
+    @include mdc-theme-dark {
+      background: #212121;
```

Comment:
Not sure we want to hard-code BG colors here, as this gives the user a lot less thematic control and assumes that this is the color of the background we want to use for dark themes. I'd take an approach [similar to dialog](https://github.com/material-components/material-components-web/tree/master/packages/mdc-dialog#theming---dark-theme-considerations) here ([source](https://github.com/material-components/material-components-web/blob/master/packages/mdc-dialog/mdc-dialog.scss#L22)).